toppra/dracula/topp_dracula_interface.py

```diff
@@ -96,17 +99,18 @@ def RunTopp(knots_ext, vlim, alim,robot_command_rate, return_spline_parameters =
     pc_acc = constraint.JointAccelerationConstraint(
         alim, discretization_scheme=constraint.DiscretizationType.Interpolation)
 
+    # gridptsnum = 10*N_samples
+    #gridpoints = np.linspace(0, 1, gridptsnum)
     # Setup a parametrization instance with hot-qpOASES
-    instance = algo.TOPPRA([pc_vel, pc_acc], path, gridpoints=np.linspace(0, 1, int(topp_breaks_count)),
-                           solver_wrapper='hotqpoases')
+    instance = algo.TOPPRA([pc_vel, pc_acc], path, gridpoints=np.linspace(0, 1, int(topp_breaks_count)),solver_wrapper='seidel')
     # print("yay we made an instance")
-    X = instance.compute_feasible_sets()
-    K = instance.compute_controllable_sets(0, 0)
+    # X = instance.compute_feasible_sets()
+    # K = instance.compute_controllable_sets(0, 0)
 
-    _, sd_vec, _ = instance.compute_parameterization(0, 0)
+    # _, sd_vec, _ = instance.compute_parameterization(0, 0)
 
-    X = np.sqrt(X)
-    K = np.sqrt(K)
+    # X = np.sqrt(X)
+    # K = np.sqrt(K)
 
     # plt.plot(X[:, 0], c='green', label="Feasible sets")
```

Comment:
Seems like this plotting code is useless now, since you have plotting code in ZeroAccelerationAtStartAndEnd.  I would either take it out, move the old code into another function, or have a flag for the similar to debug

Reply:
added it into a debug piece